packages/office-ui-fabric-react/lintCustom/customImportBlacklistRule.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+var __extends =
```

Comment:
@ecraig12345 @kenotron according to the tslint guidelines you need to generate a .js file of the .ts file with your rule. I used the regular command `npx tsc filename` to get it generated but I welcome any feedback about the proper way of doing it and the location of saving the rule if we are ok with implementing it. I scoped it to only OUFR package for now, hence the location.

Reply:
We could potentially put it in tslint-rules and set up basic build steps for that project. Though that would require either not running tslint on the lint rule or having a circular dependency...so maybe adding a new `tslint-custom-rules` package is better? Or @jdhuntington do you have any ideas? (Ken is on vacation unfortunately.)

Reply:
@ecraig12345 I looked into it a little and I think I should actually move it up into [`tslint-microsoft-contrib`](https://github.com/microsoft/tslint-microsoft-contrib) project. We are already using it so it’s probably worth it waiting until I perfect the rule and submit a PR into that project instead and then just use it in our repo. So for now I am closing the PR and rely on the built-in `import-blacklist` rule that gives us some degree of protection for now.

Reply:
Nice, this does seem like it would be a useful rule for others!